.github/workflows/codeql-analysis.yml

```
@@ -14,16 +14,15 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ master ]
+    branches: [master]
   schedule:
-    - cron: '41 6 * * 6'
+    - cron: "41 6 * * 6"
```

I don't have the crontab format memorized and I believe GH scheduled actions are UTC-based, so would it be helpful to add a comment like:
```suggestion
    # Make a pass every Saturday at 06:41 UTC
    - cron: "41 6 * * 6"
```